modules/options/src/main/scala/scala/build/options/BuildOptions.scala

```diff
@@ -595,8 +596,8 @@ object BuildOptions {
           currentEnv.keys.find(_.equalsIgnoreCase(name)).getOrElse(name)
         else
           name
-      val javaHomeKey = keyFor("JAVA_HOME")
-      val pathKey     = keyFor("PATH")
+      val javaHomeKey = keyFor(EnvsUtil.EnvVar.Java.javaHome.name)
```

Comment:
```suggestion
      val javaHomeKey = keyFor(EnvVars.Java.javaHome.name)
```
Would be much cleaner I feel

Reply:
should be better now.